### PR TITLE
Fix cron time for nightly

### PR DESCRIPTION
**Pull request by c-du**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1191,7 +1191,7 @@ workflows:
   nightly:
     triggers:
       - schedule:
-          cron: "* 18 * * *"
+          cron: "18 11 * * *"
```
Comment (RTann) [inline .circleci/config.yml:1194]: Should these be flipped for "nightly"? https://www.unix.com/man-page/POSIX/1posix/crontab/

**Review by RTann — APPROVED** [review]
Approving because it's the right idea, but I think we should flip the two numbers

File: .circleci/config.yml (hunk 1 above)
Comment (c-du) [reply .circleci/config.yml:1194]: This is early morning in UTC. So I want result ready every morning.

Comment (RTann) [reply .circleci/config.yml:1194]: I forgot it's UTC :)

**Pull request merged**